File: src/URI/URI_1073.java
package URI;
import java.util.Scanner;

public class URI_1073 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int n = sc.nextInt();
		for(int i = 2; i <= n; i += 2) {
			System.out.println(i + "^2 = " + i * i);			
		}
		
		
		sc.close();

	}

}

File: kotlin/URI_1073.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    val n = sc.nextInt()
    for (i in 2..n) {
        if (i % 2 == 0) {
            println("$i^2 = ${i*i}")
        }
    }
}
